internal/cli/models.go

```diff
@@ -486,6 +489,75 @@ type (
 	}
 )
 
+// Custom UnmarshalJSON for BlueprintProperty to capture dynamic fields
+func (bp *BlueprintProperty) UnmarshalJSON(data []byte) error {
+	// Define an alias to avoid infinite recursion
+	type Alias BlueprintProperty
+	
+	// First, unmarshal into the alias to populate known fields
+	aux := &struct {
+		*Alias
+	}{
+		Alias: (*Alias)(bp),
+	}
+	
+	if err := json.Unmarshal(data, aux); err != nil {
+		return err
+	}
+	
+	// Now unmarshal into a map to capture all fields
+	var all map[string]interface{}
+	if err := json.Unmarshal(data, &all); err != nil {
+		return err
+	}
+	
+	bp.UnknownFields = make(map[string]interface{})
+	
+	// List of known fields that shouldn't go into UnknownFields
+	knownFields := map[string]bool{
```

Comment:
I think it's better to use reflection here instead of hard coding the list of known fields